Jenkinsfile

```diff
@@ -81,6 +81,23 @@ pipeline {
                          buildDocs()
                      }
                 }
+                // stage('Build and test gcc-7 ops container') {
```

Comment:
this is now suddenly commented out

Reply:
Yes, it was a test. I will undo this now and commit again.